test/e2e/util/pod_exec.go

```diff
@@ -84,7 +87,10 @@ func (p *PodExecOptions) Validate() error {
 }
 
 // Run executes a validated remote execution against a pod.
-func (p *PodExecOptions) Run() error {
+func (p *PodExecOptions) Run(ctx context.Context) error {
+	if p.Timeout == 0 {
```

Comment:
can we just specify a const `defaultPodExecTimeout` to 2m rather that have the user explicitly pass 0 and take that as "use default"?